lighthouse-core/runner.js

```diff
@@ -52,10 +52,13 @@ class Runner {
       // that have been listed in the config.
       const filteredPasses = config.passes.map(pass => {
         pass.gatherers = pass.gatherers.filter(gatherer => {
+          if (typeof gatherer !== 'string') {
```

Comment:
(just to be clear, I realize this is just doing the same thing `instantiateGatherers` is doing in driver/index.js above, so this is likely the right hotfix, but it's a weird thing to do in `instantiateGatherers` too :)